cves/kernel/CVE-2018-10124.yml

```diff
@@ -235,10 +231,10 @@ interesting_commits:
       * Other commits that fixed a similar issue as this vulnerability
       * Anything else you find interesting.
   commits:
-  - commit:
-    note:
-  - commit:
-    note:
+  - commit: a1b2c3d4e5f67890g1234567h8901234i567890j
```

Comment:
These don't fit the format of a git hash. For the assignment, just put a link to these commits in the pull request comments and we'll figure it out.